osu.Game/Overlays/Profile/Header/Components/RankGraph.cs

```diff
@@ -65,7 +66,7 @@ protected override object GetTooltipContent(int index, int rank)
 
             return new TooltipDisplayContent
             {
-                Rank = $"#{rank:N0}",
+                Rank = rank.ToLocalisableString("\\##,##0"),
```

Comment:
Intentional change?

Reply:
It doesn't look to be localisable in its previous form, unless there's some way to prepend a `#` while still using the `N0` format.

Reply:
https://github.com/ppy/osu-web/blob/2574ce3139557b6cc5acd468c7a7ad4abc40670b/resources/assets/coffee/react/profile-page/rank.coffee#L16-L19 Hmm I suppose web does the same thing.